cypress/e2e/end-end/automatedtestsequence_hurricanemodule.spec.js

```diff
@@ -0,0 +1,192 @@
+} from "../../support/elements/student_home_page_elements";
+import * as adminHelper from "../../support/helpers/adminHelper";
+import teacherHomePageElements from "../../support/elements/teacher_home_page_elements";
+import assignmentsPageElements from "../../support/elements/assignments_page_elements";
+import {automatedtestSequenceHurricaneModuleData} from "../../support/testdata/testdata_automatedtestsequence_hurricanemodule";
+import * as LaraSequenceHelper from '../../support/helpers/laraSequenceHelper';
+import * as ReportHelper from '../../support/helpers/genericReportHelper';
+import laraSequenceElements from "../../support/elements/lara_sequence_elements";
+import * as laraRuntimeHelper from "../../support/helpers/laraRuntimeHelper";
+
+const CLASS_WORD = C.CLASS_WORD;
+const CLASS_NAME = 'Class '+ CLASS_WORD;
+const ASSIGNMENT_NAME = 'Cypress_Automated_Hurricane_Module';
+const STUDENTS = ["STUDENT1"];
+
+
+context("Verify Student Activity Work Flow", () => {
+
+    before(function() {
+        cy.visit(C.LEARN_PORTAL_BASE_URL); // Visit LEARN Portal home page
+    });
+
+    it("Edit portal settings to open activity in same browser tab", () => {
+        cy.login(C.ADMIN_USERNAME, C.ADMIN_PASSWORD);
+        adminHelper.disableOpenInNewWindow(ASSIGNMENT_NAME);
+        cy.logout();
+    });
+
+    it("Verify teacher adds class, assignment and 5 students to class", () => {
+        cy.login(C.TEACHER1_USERNAME, C.TEACHER1_PASSWORD);
+        TeacherHelper.addClass(CLASS_NAME, CLASS_NAME, CLASS_WORD);
+        TeacherHelper.addAssignment(CLASS_NAME, ASSIGNMENT_NAME);
+        TeacherHelper.openStudentRosterSection(CLASS_NAME);
+        STUDENTS.forEach(eachStudent => {
+            let studentObj = {
+                username: C[eachStudent + '_USERNAME'],
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            TeacherHelper.addRegisteredStudentToClass(studentObj.username, studentObj.firstName, studentObj.lastName, CLASS_NAME);
+        });
+
+        cy.logout();
+    });
+
+    it("Run the assignment with students added to the class", () => {
+        let totalActivitiesInSequence = automatedtestSequenceHurricaneModuleData.totalActivities;
+        let studentIndex = 0;
+        STUDENTS.forEach(eachStudent => {
+            let studentDetails = {
+                username: C[eachStudent + '_USERNAME'],
+                firstName: C[eachStudent + '_FIRSTNAME'],
+                lastName: C[eachStudent + '_LASTNAME'],
+                password: C[eachStudent + '_PASSWORD'],
+            }
+            studentIndex++;
+            cy.login(studentDetails.username, studentDetails.password);
+            BTN_ACTIVITY_RUN(CLASS_NAME).click();
+            for(let activityIndex = 2 ; activityIndex <= totalActivitiesInSequence - 4; activityIndex++){
+                cy.log("For Loop Inside");
```

Comment:
@saravanan-udayakumar , can we please remove this cy.log()? It was probably added for debugging.